aspnetcore/client-side/spa-services.md

```diff
@@ -58,7 +58,7 @@ Collectively, these infrastructure components enhance both the development workf
 ## Prerequisites for using SpaServices
 
 To work with SpaServices, install the following:
-* [Node.js](https://nodejs.org/) (version 6 or later) with npm
+* [Node.js](https://nodejs.org/en/) (version 6 or later) with npm
```

Comment:
Let's stick to the original link here. We don't want to assume a language of "en". This should be determined by the browser settings.